### PR TITLE
Add optional utm fields in Interest

### PR DESCRIPTION
**Pull request by lucas-tulio**
Adds a mutation to include `campaign`, `medium`, `source` and their initial values to the `Interest` model.

Also includes these same fields in the `createInterest` mutation input (they're optional).

### CODE REVIEW

**Comment by rhnonose** [comment]
Does this information need to go to salesforce?
If it does, we need to modify `buyer_leads` to be sent.

**Comment by lucas-tulio** [comment]
@rhnonose No need 👍

File: apps/re_web/test/graphql/interests/mutation_test.exs [hunk 1]
```diff
@@ -29,6 +29,12 @@ defmodule ReWeb.GraphQL.Interests.MutationTest do
         "email" => "testemail@emcasa.com",
         "phone" => "123321123",
         "message" => "this website is cool",
+        "campaign" => "utm campaign",
+        "medium" => "utm medium",
+        "source" => "utm source",
+        "initial_campaign" => "utm initial_campaign",
+        "initial_medium" => "utm initial_medium",
+        "initial_source" => "utm initial_source",
```
Comment (rhnonose) [inline apps/re_web/test/graphql/interests/mutation_test.exs]: These should be camel cased.

File: apps/re_web/test/graphql/interests/mutation_test.exs [hunk 2]
```diff
@@ -59,6 +71,12 @@ defmodule ReWeb.GraphQL.Interests.MutationTest do
              "email" => "testemail@emcasa.com",
              "phone" => "123321123",
              "message" => "this website is cool",
+             "campaign" => "utm campaign",
+             "medium" => "utm medium",
+             "source" => "utm source",
+             "initial_campaign" => "utm initial_campaign",
+             "initial_medium" => "utm initial_medium",
+             "initial_source" => "utm initial_source",
```
Comment (rhnonose) [inline apps/re_web/test/graphql/interests/mutation_test.exs]: These should be camel cased.

File: apps/re_web/test/graphql/interests/mutation_test.exs [hunk 3]
```diff
@@ -83,6 +101,12 @@ defmodule ReWeb.GraphQL.Interests.MutationTest do
         "email" => "testemail@emcasa.com",
         "phone" => "123321123",
         "message" => "this website is cool",
+        "campaign" => "utm campaign",
+        "medium" => "utm medium",
+        "source" => "utm source",
+        "initial_campaign" => "utm initial_campaign",
+        "initial_medium" => "utm initial_medium",
+        "initial_source" => "utm initial_source",
```
Comment (rhnonose) [inline apps/re_web/test/graphql/interests/mutation_test.exs]: These should be camel cased.

File: apps/re_web/test/graphql/interests/mutation_test.exs [hunk 4]
```diff
@@ -113,6 +143,12 @@ defmodule ReWeb.GraphQL.Interests.MutationTest do
              "email" => "testemail@emcasa.com",
              "phone" => "123321123",
              "message" => "this website is cool",
+             "campaign" => "utm campaign",
+             "medium" => "utm medium",
+             "source" => "utm source",
+             "initial_campaign" => "utm initial_campaign",
+             "initial_medium" => "utm initial_medium",
+             "initial_source" => "utm initial_source",
```
Comment (rhnonose) [inline apps/re_web/test/graphql/interests/mutation_test.exs]: These should be camel cased.

**Pull request merged**